src/pykiso/interfaces/thread_auxiliary.py

```diff
@@ -68,7 +69,7 @@ def __init__(
         """
         # Initialize thread class
         super().__init__()
```

Comment:
Just learned that we do this wrong! -> https://stackoverflow.com/questions/9575409/calling-parent-class-init-with-multiple-inheritance-whats-the-right-way

We should actually:
```suggestion
        threading.Thread.__init__()
        AuxiliaryCommon.__init__()
```

This also explain why the coverage of the auxiliary is so low compared to the others. @BKaDamien, can you somehow confirm this?